app/views/shared/authentication/_forem_creator_signup.html.erb

```diff
@@ -92,7 +92,13 @@
         <% else %>
           <div class="crayons-field mt-6">
             <%= f.label :forem_owner_secret, t("views.auth.register.secret.label"), class: "crayons-field__label" %>
-            <%= f.password_field :forem_owner_secret, class: "crayons-textfield", required: ForemInstance.needs_owner_secret?, placeholder: t("views.auth.register.secret.placeholder") %>
+            <div class="relative js-forem_owner_secret-toggle-wrapper">
```

Comment:
```suggestion
            <div class="relative js-forem-owner-secret-toggle-wrapper">
```
nitpick (not-blocking) - I understand why it's this way, but we try to avoid mixing casing in classnames, as it's just a bit prone to mis-typing and error - can we change references to this class to kebab case liked above?

Reply:
A kebab case would be appropriate here!
Fixed!
c004c0f2ea9b1998802fa376bc456f698bcfbe75

Reply:
@aitchiss 
Fixed a bug that occurred when the "New Forem Secret" form did not exist.

Rather than adding a conditional statement to check for the existence of a form, we thought it would be simpler to add an event for the shared class.
[2aeaedb](https://github.com/forem/forem/pull/17016/commits/2aeaedbb387f1f0a47ba19031b304e118b08be53)

Reply:
Yes I saw that in the latest commit - I think the approach makes sense. It also means this code is easier to extract into a utility/helper at a later time if we begin to need it in other views too 👍